doc/rados/operations/control.rst

```diff
@@ -162,16 +166,16 @@ Remove the given OSD(s). ::
 
 	ceph osd rm [{id}...]
 
-Query the current max_osd parameter in the OSD map. ::
+Query the current ``max_osd`` parameter in the OSD map. ::
 
 	ceph osd getmaxosd
 
 Import the given crush map. ::
 
 	ceph osd setcrushmap -i file
 
-Set the ``max_osd`` parameter in the OSD map. This is necessary when
-expanding the storage cluster. ::
+Set the ``max_osd`` parameter in the OSD map. This in rare occasions necessary when
```

Comment:
Ack. I didn't even know this was a command, since there's a config file option with a different name, and the current default value is sufficient for mortals.  I've reworded the whole thing.